client/asset/eth/rpcclient_harness_test.go

```diff
@@ -6,7 +6,10 @@
 // These tests are expected to be run in descending as some depend on the tests before. They cannot
 // be run in parallel.
 //
-// NOTE: Occationally tests will fail with "timed out". Please try again...
+// TODO: Occasionally tests will fail with "timed out" because transactions are
+// not being mined. If one tx is not mined, following txs from the same account
+// with a higher nonce also cannot be mined, so it causes all tests after to
+// fail as well. Find out why the first tx fails to be broadcast.
```

Comment:
This seems to be unrelated to  #1130

Reply:
Possibly realated...

https://github.com/ethereum/go-ethereum/issues/21385

I've spent a bit of time on this, but unable to stop this from happening so far.

Reply:
Headway on this. After adding better logging, the server was telling us: 
`TRACE[07-28|11:37:15.770] Discarding invalid transaction           hash=5a1f01..40f83e err="nonce too low"`

Which suggests that the light client sometimes has trouble discerning the correct nonce. Probably because we destroy it's persistent data every test suite.